src/helpers/config.ts

```diff
@@ -54,8 +58,12 @@ const getNetlifyRoutes = (nextRoute) => {
   return netlifyRoutes
 }
 
-exports.generateRedirects = async ({ netlifyConfig, basePath, i18n }) => {
-  const { dynamicRoutes, routes: staticRoutes } = await readJSON(
+export const generateRedirects = async ({ netlifyConfig, basePath, i18n }: { 
+  netlifyConfig: NetlifyConfig, 
+  basePath: string, 
+  i18n 
```

Comment:
You can get this type by importing the `NextConfig` type from `next`

Reply:
oo thanks! Wasn't sure where it was coming from